<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fondo</class>
 <widget class="QWidget" name="Fondo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Fondo{
	background:#00BCD4;
}

#barratitulo{
	background:#fff;
}

#titulo{
	color :#263238;
}

#groupBox{
	background:#000
}

#lblId, #lblNombre, #lblApellido,  #lblCorreo, #lblTelefono, #lblDireccion{
	color:#fff;
 	font-size: 12px;
  	font-family: Arial, Verdana;
}

.QLineEdit{
	 border: 1px solid #DBE1EB;
  	 font-size: 11px;
  	font-family: Arial, Verdana;
}
</string>
  </property>
  <widget class="QFrame" name="barratitulo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aplicación Básica con Python</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="frmUsuarios">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblId">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtId"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblNombre">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtNombre"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblApellido">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtApellido"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblCorreo">
      <property name="text">
       <string>Correo</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtCorreo"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblTelefono">
      <property name="text">
       <string>Teléfono</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtTelefono"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblDireccion">
      <property name="text">
       <string>Dirección</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtDireccion"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMostrar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="listaClientes2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>641</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telefono</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dirección</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
